src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,19 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[3];
+
+        machines[0] = new Truck();
+        machines[1] = new Bulldozer();
+        machines[2] = new Excavator();
+
+        for (Machine mc : machines) {
+            mc.doWork();
+            mc.stopWork();
+            System.out.println();
+        }
```

Comment:
Please avoid using 1-2 letter shortened naming, it doesn't really have any benefits but may cause problems with code readability
```suggestion
        for (Machine machine : machines) {
            machine.doWork();
            machine.stopWork();
            System.out.println();
        }
```

Reply:
@olekskov I took into consideration your advice and fixed the code. Created a new branch  Jv oop test new with two commits, I will be grateful if you check test_two. I also have a problem with GIT, because I don't understand how to make a Pull request so that it is displayed in yellow on the MateAcademy website, but after correcting the code, maybe you can give me some advice because, in my opinion, it isn't a very good idea to create a new branch to check the same code if I can do the same in the same branch or maybe is better do only one other commit in the same branch. I don't know maybe you give me advice on how can I do it better?